lib/src/validators/validators.dart

```diff
@@ -142,20 +142,20 @@ class Validators {
     return ComposeOrValidator(validators).validate;
   }
 
-  /// Gets a validator that requires the control's value list to
-  /// contain all the values provided [contain].
+  /// Gets a validator that requires the control's value contains all the
+  /// values specified in [values].
   ///
-  /// The argument [valuesToCompare] must not be null.
+  /// The argument [values] must not be null.
   ///
   /// ### Example:
   /// Validates that 'list' contains all the items provided
   /// ```dart
   /// final control = FormControl<List<int>>(
-  ///   'value': [1,2,3,4,5,6],
-  ///   'validators': [Validators.contain([1,3,5])],
+  ///   'value': [1,2,3],
+  ///   'validators': [Validators.contain([1,3])],
```

Comment:
the name of the method is **Validators.contains**, but in the example is missing the **s**